test/pipeline-steps.test.ts

```diff
@@ -73,6 +73,15 @@ describe("extractFilesetFromPath", () => {
     ).toBe(false);
   });
 
+  test("should return manifest file when given manifest file path directly", () => {
+    const manifestPath =
+      "test/fixtures/manifest-only-package/Manifest.ocf.json";
```

Comment:
```suggestion
    const manifestPath =
      fixturePath("manifest-only-package/Manifest.ocf.json");
```

:wrench: Minor, but use the helper function.